functional-tests/pages/recaptcha.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+const { BasePage } = require('kakunin');
+
+class RecaptchaPage extends BasePage {
+  constructor() {
+    super();
+
+    this.url = '/recaptcha';
+
+    this.form = $('#comment_form')
+    this.emailElement = $('input[type=email]')
+    this.postCommentButton = $('input[name=submit]');
+    this.recaptchaNotConfirmed = $('pre')
```

Comment:
is it correct selector?

Reply:
probably, but at the same time it's too weak